### PR TITLE
[WIP] DNS: update for track 2 migration

### PR DESCRIPTION
**Pull request by xccc-msft**

### CODE REVIEW

File: azure-mgmt-dns/pom.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-mgmt-resources</artifactId>
+      <version>2.0.0-SNAPSHOT</version>
```
Comment (yungezz) [inline azure-mgmt-dns/pom.xml]: why this one version is not managed in parent?

File: azure-mgmt-dns/src/main/java/com/azure/management/dns/CloudError.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+package com.azure.management.dns;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The CloudError model.
+ */
+@Fluent
+public final class CloudError {
```
Comment (yungezz) [inline azure-mgmt-dns/src/main/java/com/azure/management/dns/CloudError.java]: still CloudError?

Comment (weidongxu-microsoft) [reply azure-mgmt-dns/src/main/java/com/azure/management/dns/CloudError.java]: @xccc-msft You can just delete it. Generator will no longer generate this class.

File: azure-mgmt-dns/pom.xml (hunk 1 above)
Comment (xccc-msft) [reply azure-mgmt-dns/pom.xml]: No we cannot manage in parent pom as this is not external dependency.

File: azure-mgmt-dns/src/main/java/com/azure/management/dns/CloudError.java (hunk 2 above)
Comment (xccc-msft) [reply azure-mgmt-dns/src/main/java/com/azure/management/dns/CloudError.java]: Deleted.

**Pull request merged**